packages/patternfly-4/react-core/src/experimental/components/index.ts

```diff
@@ -1,5 +1,5 @@
 export * from './DataToolbar';
-export * from './Divider';
+export * from '../../components/Divider';
 export * from './Drawer';
-export * from './Spinner';
```

Comment:
Would it be possible to deprecate the spinner (e.g., for at least one milestone) from the experimental package Vs removing it immediately?

We're using the spinner in Cost Management and this change would break us. It would give folks a little warning that apps must update.

Reply:
Never mind, I see that spinner is in the index file.